features/keystone/identity/ldap.pan

```diff
@@ -9,18 +9,14 @@ include 'components/metaconfig/config';
 prefix '/software/components/metaconfig/services/{/etc/keystone/keystone.conf}';
 'contents/identity/domain_specific_drivers_enabled' = 'True';
 'contents/identity/domain_config_dir' = '/etc/keystone/domains';
-'contents/identity/driver' = 'ldap';
+'contents/identity/driver' = 'sql';
 'contents/resource/driver' = 'sql';
 'contents/assignment/driver' = 'sql';
 
 # All ldap configuration is put on /etc/keystone/domains/keystone.DOMAIN_NAME.conf
 prefix '/software/components/metaconfig';
 'services' = {
-  # default domain is still on SQL
```

Comment:
Why is that a problem, no configuration file for the default domain needs to be created in the keystone/domains/ folder as the only configuration there is the keystone default and defining this in both causes more problems?
Only if additional config is required should the file be created.